style-guide.md

```diff
@@ -0,0 +1,585 @@
+Archivematica documentation consists of three manuals:
+
+* [Getting started](https://www.archivematica.org/en/docs/archivematica-1.7/#getting-started)
+* [Administrator manual](https://www.archivematica.org/en/docs/archivematica-1.7/#administrator-manual)
+* [User manual](https://www.archivematica.org/en/docs/archivematica-1.7/#user-manual)
+
+These manuals correspond to three top-level directories in the
+[documentation repository](https://github.com/artefactual/archivematica-docs):
+
+* [_getting-started_](https://github.com/artefactual/archivematica-docs/tree/1.7/getting-started)
+* [_admin-manual_](https://github.com/artefactual/archivematica-docs/tree/1.7/admin-manual)
+* and [_user-manual_](https://github.com/artefactual/archivematica-docs/tree/1.7/user-manual).
+
+The manuals are meant to be stable. Changes at this level may also require
+layout changes to the documentation website. New top-level directories should
+not be added unless there is consensus with Artefactual staff to make a major
+change.
+
+### Chapters
+
+Each manual contains chapters, which are represented in the repository as
+subdirectories within a manual. For example, the [_User
+manual_](https://github.com/artefactual/archivematica-docs/tree/1.7/user-manual)
+has a chapter called
+[_transfer_](https://github.com/artefactual/archivematica-docs/tree/1.7/user-manual/transfer).
+Within _Transfer_ are individual reStructuredText (reST) files, which represent
+pages, as well as a directory of images that are referenced by the reST files. There may be other reference directories as well.
+
+Chapters generally correspond to tabs in the interface (such as _Transfer_ and
+_Ingest_) or topics that are in some way greater than the interface (such as
+_Metadata_ or _Translations_). Be thoughtful about adding a new chapter. Most
+features occur within an already-defined workflow. Here are two examples for
+why a new chapter might be added:
+
+* A new tab is added to the Archivematica dashboard. For example, when the
+Appraisal tab was introduced in Archivematica 1.6, a chapter titled
+[_appraisal_](https://github.com/artefactual/archivematica-docs/tree/1.7/user-manual/appraisal)
+was added to the User manual as a place to record appraisal workflow
+instructions.
+* A feature is introduced that has implications outside normal user workflows.
+For example, when internationalization hooks were added to the code, a chapter
+titled
+[_translations_](https://github.com/artefactual/archivematica-docs/tree/1.7/user-manual/translations)
+was added to the User Manual as a place to put translation instructions.
+
+Though somewhat less strictly enforced than the creation of a new manual, new
+chapters should only be created if there is consensus with Artefactual staff.
+
+Chapters can contain sub-directories if needed. For example, the [_installation
+and
+setup_](https://github.com/artefactual/archivematica-docs/tree/1.7/admin-manual/installation-setup)
```

Comment:
You can also have named anchors in Markdown to make text wrappings easier:

```
This is a [link][google] to show 

[google]: https://www.google.ca
```

Reply:
:astonished: